pkgs/applications/networking/browsers/chromium/browser.nix

```diff
@@ -12,11 +12,10 @@ mkChromiumDerivation (base: rec {
   sandboxExecutableName = "__chromium-suid-sandbox";
 
   installPhase = ''
-    mkdir -p "$libExecPath/swiftshader"
+    mkdir -p "$libExecPath/chromium"
```

Comment:
Just `$libExecPath` is enough, because it [already contains `chromium`](https://github.com/bendlas/nixpkgs/blob/7b866fdff90c6400e3c82c45e7390adc4563eb91/pkgs/applications/networking/browsers/chromium/common.nix#L86), the reason this was `$libExecPath/swiftshader` is that `mkdir -p` already creates parent directories, which then includes `$out/libexec/chromium`.

Reply:
In my last build, I had shortened it to `mkdir -p "$libExecPath", and the `installPhase` broke due to that directory missing. I changed it to this version, `nix-shell`ed into the `--keep-failed` chromium build, ran `out=/tmp/out; sandbox=/tmp/sandbox eval "$configurePhase"; eval "$buildPhase"; eval "$installPhase"; fixupPhase` and got, what looked like a good build in `/tmp/out`. I'm re-running build now, at `[21185/27987]` to verify. I'll probably check back tomorrow.

Reply:
oh .. I only saw `already contains chromium` now .. that seems fishy ...

Reply:
you're right, I probably interpreted the build error wrong, and it actually broke because of the spurious ` `, I had left in before: https://github.com/NixOS/nixpkgs/pull/29335/commits/209b9e88df6c9802185cfe004d821dfcaf1992ad#diff-69a3af1d07b37077e7dd11ccc8f9b61eR15